src/devices/tfa_14_1504.c

```diff
@@ -0,0 +1,199 @@
+/** @file
+    Decoder for TFA Dostmann 14.1504
+    Radio-controlled grill and meat thermometer
+
+    Copyright (C) 2022-2023 Joël Bourquard <joel.bourquard@gmail.com>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+
+/**
+Decoder for temperature sensor model no. TPX305
```

Comment:
This should include (replicate) line 2 infos.

Reply:
Done! Thanks